presto-native-execution/presto_cpp/presto_protocol/Connectors.cpp

```diff
@@ -44,6 +44,10 @@ void registerHiveConnectors() {
   registerConnector("hive-hadoop2", "hive");
 }
 
+void registerIcebergConnector() {
+  registerConnector("iceberg", "iceberg");
```

Comment:
Let's keep this as is, but add a comment saying the connectorName and connectorKey are for Presto, not Presto native.

Reply:
@yingsu00 I have made the change as we had discussed earlier and moved the iceberg registration as part of the hive connectors